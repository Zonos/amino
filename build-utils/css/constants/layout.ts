export const layoutStyles = {
  'appbar-height': '55px',
  'sidebar-width': '264px',
  'sidebar-color': 'white',
  'header-color': 'white',
  'backdrop-color': 'var(--amino-gray-900)',
  'elevation-0': '0',
  'elevation-100': '100',
  'elevation-200': '200',
  'elevation-300': '300',
  'elevation-400': '400',
  'elevation-500': '500',
  'dialog-elevation': 'var(--amino-elevation-400)',
  'toast-elevation': 'var(--amino-elevation-500)',
  'appbar-elevation': 'var(--amino-elevation-500)',
} as const;

export const shadows = {
  'v3-shadow-inset': 'inset 0px 2px 4px rgba(0, 0, 0, 0.06)',
  'v3-shadow-base': `0px 1px 2px rgba(0, 0, 0, 0.06), 0px 1px 3px rgba(0, 0, 0, 0.1)`,
  'v3-shadow-medium': `0px 2px 4px rgba(0, 0, 0, 0.06), 0px 4px 6px rgba(0, 0, 0, 0.1)`,
  'v3-shadow-large': `0px 10px 15px rgba(0, 0, 0, 0.1), 0px 4px 6px rgba(0, 0, 0, 0.05)`,
  'v3-shadow-xl': `0px 20px 25px rgba(0, 0, 0, 0.1), 0px 10px 10px rgba(0, 0, 0, 0.04)`,
  'v3-shadow-xxl': '0px 32px 50px rgba(0, 0, 0, 0.24)',
  'shadow-inset': 'var(--amino-v3-shadow-inset)',
  'shadow-small': 'var(--amino-v3-shadow-base)',
  'shadow-base': 'var(--amino-v3-shadow-medium)',
  'shadow-medium': 'var(--amino-v3-shadow-large)',
  'shadow-large': 'var(--amino-v3-shadow-xl)',
  'shadow-larger': 'var(--amino-v3-shadow-xxl)',
} as const;

export const borders = {
  border: '1px solid var(--amino-border-color)',
  'border-blue': '1px solid var(--amino-blue-400)',
  'border-red': '1px solid var(--amino-red-300)',
  'border-cyan': '1px solid var(--amino-cyan-300)',
  'border-orange': '1px solid var(--amino-orange-300)',
  'border-green': '1px solid var(--amino-green-300)',
  'border-purple': '1px solid var(--amino-purple-300)',
  'border-amber': '1px solid var(--amino-amber-300)',
  'border-transparent': '1px solid transparent',
} as const;

export const layoutStyleList = {
  ...layoutStyles,
  ...shadows,
  ...borders,
};
