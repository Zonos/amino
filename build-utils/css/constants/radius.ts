export const radius = {
  /** @info 4px */
  'radius-sm': '0.25rem',
  /** @info 6px */
  radius: '0.375rem',
  /** @info 8px */
  'radius-lg': '0.5rem',
  /** @info 12px */
  'radius-xl': '0.75rem',
} as const;
