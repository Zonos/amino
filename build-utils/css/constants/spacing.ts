export const spaces = {
  /** @info px: 24px */
  space: '1.5rem',
  /** @info px: -24px */
  'space-negative': '-1.5rem',
  /** @info px: 16px */
  'space-half': '1rem',
  /** @info px: -16px */
  'space-half-negative': '-1rem',
  /** @info px: 8px */
  'space-quarter': '0.5rem',
  /** @info px: -8px */
  'space-quarter-negative': '-0.5rem',
  /** @info px: 40px */
  'space-double': '2.5rem',
  /** @info px: -40px */
  'space-double-negative': '-2.5rem',
} as const;
