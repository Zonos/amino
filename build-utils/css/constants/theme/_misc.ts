export const misc = {
  'disabled-opacity': '0.6',
};
