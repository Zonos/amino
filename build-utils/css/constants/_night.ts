import { theme } from './theme';
import { blue } from './theme/night/colors/_blue';
import { cyan } from './theme/night/colors/_cyan';
import { gray } from './theme/night/colors/_gray';
import { red } from './theme/night/colors/_red';
import { orange } from './theme/night/colors/_orange';
import { green } from './theme/night/colors/_green';
import { purple } from './theme/night/colors/_purple';
import { transparent } from './theme/night/colors/_transparent';

import { constraintDefinedAminoVar } from './utils';

export const night = constraintDefinedAminoVar(theme, {
  /* GRAY PALETTE */
  ...gray,

  /* BLUE PALETTE */
  ...blue,

  /* CYAN PALETTE */
  ...cyan,

  /* RED PALETTE */
  ...red,

  /* ORANGE PALETTE */
  ...orange,

  /* GREEN PALETTE */
  ...green,

  /* PURPLE */
  ...purple,

  /* TRANSPARENT */
  ...transparent,

  'page-background': 'var(--amino-gray-50)',
  'surface-color': 'var(--amino-gray-base)',
  'surface-color-secondary': 'var(--amino-gray-50)',
  'border-color': 'var(--amino-gray-l60)',
  'text-color': 'white',
  'input-background': 'var(--amino-gray-l40)',
  'hover-color': 'var(--amino-gray-l20)',

  'v3-shadow-inset': 'inset 0px 2px 4px rgba(0, 0, 0, 0.18)',
  'v3-shadow-base':
    '0px 1px 2px rgba(0, 0, 0, 0.18), 0px 1px 3px rgba(0, 0, 0, 0.3)',
  'v3-shadow-medium':
    '0px 2px 4px rgba(0, 0, 0, 0.18), 0px 4px 6px rgba(0, 0, 0, 0.3)',
  'v3-shadow-large':
    '0px 10px 15px rgba(0, 0, 0, 0.3), 0px 4px 6px rgba(0, 0, 0, 0.15)',
  'v3-shadow-xl':
    '0px 20px 25px rgba(0, 0, 0, 0.3), 0px 10px 10px rgba(0, 0, 0, 0.12)',
  'v3-shadow-xxl': '0px 32px 50px rgba(0, 0, 0, 0.72)',
  'backdrop-color': 'var(--amino-gray-l60)',
  'header-color': 'var(--amino-gray-l80)',
  border: '1px solid var(--amino-border-color)',
});
