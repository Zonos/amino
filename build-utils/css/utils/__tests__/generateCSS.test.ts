import { night } from '../../constants/_night';
import { theme } from '../../constants/theme';
import {
  generateNightThemeContent,
  generateLightThemeContent,
} from '../generateCSS';

test(`Case light theme:`, async () => {
  const result = await generateLightThemeContent(theme);
  expect(result).toMatchSnapshot();
});

test(`Case night theme:`, async () => {
  const result = await generateNightThemeContent(night);
  expect(result).toMatchSnapshot();
});
