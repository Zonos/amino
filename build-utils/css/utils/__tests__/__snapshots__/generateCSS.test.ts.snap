// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Case light theme: 1`] = `
":root,
[data-theme='day'] {
  color-scheme: light;
  --amino-transition: all 0.2s ease-in-out 0s, visibility 0s;
  --amino-type-scale-base: 16px;
  --amino-glass-0: #10111600;
  --amino-glass-50: #1011160a;
  --amino-glass-100: #10111614;
  --amino-glass-200: #10111626;
  --amino-glass-300: #1011163d;
  --amino-glass-400: #10111656;
  --amino-glass-500: #10111670;
  --amino-glass-600: #1011169b;
  --amino-glass-700: #101116ad;
  --amino-glass-800: #101116bc;
  --amino-glass-900: #101116c6;
  --amino-glass-950: #101116dd;
  --amino-glass-1000: #101116ef;
  --amino-gray-0: #ffffff;
  --amino-gray-50: #f4f5f6;
  --amino-gray-100: #e8eaee;
  --amino-gray-200: #dfe2e7;
  --amino-gray-300: #bcc3cd;
  --amino-gray-400: #8a95a8;
  --amino-gray-500: #68748d;
  --amino-gray-600: #555d72;
  --amino-gray-700: #454c5e;
  --amino-gray-800: #343846;
  --amino-gray-900: #272a35;
  --amino-gray-950: #1e2029;
  --amino-gray-1000: #0f1015;
  --amino-blue-50: #e7f0fe;
  --amino-blue-100: #dae7fc;
  --amino-blue-200: #c2d6fb;
  --amino-blue-300: #98b6f8;
  --amino-blue-400: #678ff3;
  --amino-blue-500: #446fee;
  --amino-blue-600: #2e55e2;
  --amino-blue-700: #2544d0;
  --amino-blue-800: #2438a9;
  --amino-blue-900: #232e85;
  --amino-blue-950: #1a1f51;
  --amino-cyan-50: #e6fefd;
  --amino-cyan-100: #d6fffe;
  --amino-cyan-200: #c0f2f2;
  --amino-cyan-300: #9be7e9;
  --amino-cyan-400: #66d5db;
  --amino-cyan-500: #3ec4d0;
  --amino-cyan-600: #2ca0af;
  --amino-cyan-700: #29818e;
  --amino-cyan-800: #25646f;
  --amino-cyan-900: #1d464e;
  --amino-cyan-950: #152d32;
  --amino-red-50: #ffeceb;
  --amino-red-100: #fee5e5;
  --amino-red-200: #fccfd0;
  --amino-red-300: #faa7ab;
  --amino-red-400: #f67681;
  --amino-red-500: #ee4559;
  --amino-red-600: #dd314b;
  --amino-red-700: #b81835;
  --amino-red-800: #9a1734;
  --amino-red-900: #75142d;
  --amino-red-950: #49081a;
  --amino-orange-50: #fef4e7;
  --amino-orange-100: #fdebd7;
  --amino-orange-200: #f9d3af;
  --amino-orange-300: #f5b27c;
  --amino-orange-400: #f08e47;
  --amino-orange-500: #ec6f23;
  --amino-orange-600: #e5611a;
  --amino-orange-700: #b74a17;
  --amino-orange-800: #923c1a;
  --amino-orange-900: #763018;
  --amino-orange-950: #3f160b;
  --amino-green-50: #f0faef;
  --amino-green-100: #e0f8e0;
  --amino-green-200: #c3efc5;
  --amino-green-300: #93e29a;
  --amino-green-400: #5dcb6a;
  --amino-green-500: #37b04a;
  --amino-green-600: #2ca041;
  --amino-green-700: #237234;
  --amino-green-800: #205b2f;
  --amino-green-900: #1c4b29;
  --amino-green-950: #0a2914;
  --amino-purple-50: #fcf0ff;
  --amino-purple-100: #fae8ff;
  --amino-purple-200: #f4d4ff;
  --amino-purple-300: #e9b2ff;
  --amino-purple-400: #d782fe;
  --amino-purple-500: #c052fa;
  --amino-purple-600: #a126ed;
  --amino-purple-700: #871ed2;
  --amino-purple-800: #6c1eab;
  --amino-purple-900: #4e1881;
  --amino-purple-950: #2f055c;
  --amino-transparent-white: #ffffffbb;
  --amino-transparent-black: #101116bb;
  --amino-transparent-gray-600: #6a6d771f;
  --amino-v3-shadow-base: 0px 1px 3px 0px rgba(10, 10, 11, 0.1),
    0px 1px 2px rgba(10, 10, 11, 0.06), 0px 0px 0px 1px rgba(110, 110, 120, 0.1);
  --amino-v3-shadow-medium: 0px 2px 4px 0px rgba(10, 10, 11, 0.06),
    0px 4px 6px rgba(10, 10, 11, 0.1), 0px 0px 0px 1px rgba(110, 110, 120, 0.1);
  --amino-v3-shadow-large: 0px 4px 6px 0px rgba(10, 10, 11, 0.05),
    0px 10px 15px rgba(10, 10, 11, 0.1),
    0px 0px 0px 1px rgba(110, 110, 120, 0.1);
  --amino-v3-shadow-xl: 0px 10px 10px 0px rgba(12, 12, 13, 0.04),
    0px 20px 25px rgba(10, 10, 11, 0.1),
    0px 0px 0px 1px rgba(110, 110, 120, 0.1);
  --amino-v3-shadow-xxl: 0px 32px 50px 0px rgba(10, 10, 11, 0.24),
    0px 0px 0px 1px rgba(110, 110, 120, 0.1);
  --amino-v3-shadow-inset: 0px 2px 4px rgba(0, 0, 0, 0.04),
    0px 1px 0px 0px rgba(255, 255, 255, 1),
    inset 0px 0px 8px rgba(0, 0, 0, 0.02),
    inset 0px 0px 0px 0.5px rgba(0, 0, 0, 0.06),
    inset 0px 2px 4px rgba(0, 0, 0, 0.04), inset 0px 1px 1px rgba(0, 0, 0, 0.04);
  --amino-shadow-button-primary: 0px 0px 0px 1px rgba(0, 0, 0, 0.08) inset,
    0px 1px 3px 0px rgba(68, 111, 238, 0.4),
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset;
  --amino-shadow-button-success: 0px 0px 0px 1px rgba(0, 0, 0, 0.08) inset,
    0px 1px 3px 0px rgba(55, 176, 74, 0.4),
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset;
  --amino-shadow-button-danger: 0px 0px 0px 1px rgba(0, 0, 0, 0.08) inset,
    0px 1px 3px 0px rgba(238, 69, 89, 0.4),
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset;
  --amino-shadow-button-warning: 0px 0px 0px 1px rgba(0, 0, 0, 0.08) inset,
    0px 1px 3px 0px rgba(236, 111, 35, 0.4),
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset;
  --amino-button-focus-ring: 0 0 0 2px var(--amino-blue-500);
  --amino-shadow-button-standard: 0px -1px 1px 1px rgba(0, 0, 0, 0.08) inset,
    0px 0px 0px 1px rgba(106, 109, 119, 0.12) inset,
    0px 1px 3px 0px rgba(110, 110, 120, 0.12);
  --amino-shadow-button-disabled: 0px 1px 3px 0px rgba(16, 17, 22, 0.2),
    0px -1px 1px 1px rgba(0, 0, 0, 0.08) inset;
  --amino-shadow-select-base: 0px 1px 3px 0px rgba(110, 110, 120, 0.12),
    0px -1px 1px 1px rgba(0, 0, 0, 0.08) inset,
    0px 0px 0px 1px rgba(106, 109, 119, 0.12) inset;
  --amino-shadow-select-active: 0px 1px 3px 0px rgba(68, 94, 238, 0.4),
    0px 0px 0px 1px #445eee inset,
    0px -1px 1px 1px rgba(68, 94, 238, 0.12) inset;
  --amino-shadow-select-error: 0px 1px 3px 0px rgba(221, 49, 78, 0.4),
    0px 0px 0px 1px #dd314e inset,
    0px -1px 1px 1px rgba(221, 49, 78, 0.12) inset;
  --amino-opacity-disabled: 0.6;
  --amino-space-negative-80: -5rem;
  --amino-space-negative-76: -4.75rem;
  --amino-space-negative-72: -4.5rem;
  --amino-space-negative-68: -4.25rem;
  --amino-space-negative-64: -4rem;
  --amino-space-negative-60: -3.75rem;
  --amino-space-negative-56: -3.5rem;
  --amino-space-negative-52: -3.25rem;
  --amino-space-negative-48: -3rem;
  --amino-space-negative-44: -2.75rem;
  --amino-space-negative-40: -2.5rem;
  --amino-space-negative-36: -2.25rem;
  --amino-space-negative-32: -2rem;
  --amino-space-negative-28: -1.75rem;
  --amino-space-negative-24: -1.5rem;
  --amino-space-negative-20: -1.25rem;
  --amino-space-negative-16: -1rem;
  --amino-space-negative-12: -0.75rem;
  --amino-space-negative-8: -0.5rem;
  --amino-space-negative-4: -0.25rem;
  --amino-space-0: 0;
  --amino-space-4: 0.25rem;
  --amino-space-8: 0.5rem;
  --amino-space-12: 0.75rem;
  --amino-space-16: 1rem;
  --amino-space-20: 1.25rem;
  --amino-space-24: 1.5rem;
  --amino-space-28: 1.75rem;
  --amino-space-32: 2rem;
  --amino-space-36: 2.25rem;
  --amino-space-40: 2.5rem;
  --amino-space-44: 2.75rem;
  --amino-space-48: 3rem;
  --amino-space-52: 3.25rem;
  --amino-space-56: 3.5rem;
  --amino-space-60: 3.75rem;
  --amino-space-64: 4rem;
  --amino-space-68: 4.25rem;
  --amino-space-72: 4.5rem;
  --amino-space-76: 4.75rem;
  --amino-space-80: 5rem;
  --amino-border: 1px solid var(--amino-border-color);
  --amino-border-blue: 1px solid var(--amino-blue-500);
  --amino-border-red: 1px solid var(--amino-red-500);
  --amino-border-cyan: 1px solid var(--amino-cyan-500);
  --amino-border-orange: 1px solid var(--amino-orange-500);
  --amino-border-green: 1px solid var(--amino-green-500);
  --amino-border-purple: 1px solid var(--amino-purple-500);
  --amino-border-amber: 1px solid var(--amino-amber-500);
  --amino-border-transparent: 1px solid transparent;
  --amino-border-subtle: 1px solid var(--amino-gray-100);
  --amino-sidebar-color: var(--amino-gray-0);
  --amino-header-color: var(--amino-gray-0);
  --amino-danger: var(--amino-red-500);
  --amino-danger-dark: var(--amino-red-700);
  --amino-error: var(--amino-red-500);
  --amino-primary: var(--amino-blue-500);
  --amino-primary-light: var(--amino-blue-400);
  --amino-primary-dark: var(--amino-blue-800);
  --amino-success: var(--amino-green-500);
  --amino-success-dark: var(--amino-green-700);
  --amino-warning: var(--amino-orange-500);
  --amino-warning-dark: var(--amino-orange-600);
  --amino-glow-error: 0 0 0 1px var(--amino-red-500);
  --amino-glow-blue: 0 0 0 1px var(--amino-primary);
  --amino-glow-cyan: 0 0 0 1px var(--amino-cyan-500);
  --amino-glow-red: 0 0 0 1px var(--amino-red-500);
  --amino-glow-orange: 0 0 0 1px var(--amino-orange-500);
  --amino-glow-green: 0 0 0 1px var(--amino-green-500);
  --amino-glow-purple: 0 0 0 1px var(--amino-purple-500);
  --amino-page-background: var(--amino-gray-0);
  --amino-border-color: var(--amino-gray-200);
  --amino-hover-color: var(--amino-gray-50);
  --amino-text-color: var(--amino-gray-1000);
  --amino-text-color-secondary: var(--amino-gray-700);
  --amino-focus-button-standard: 0px 0px 0px 1px
      var(--amino-transparent-gray-600) inset,
    var(--amino-button-focus-ring);
  --amino-focus-button: 0px 0px 0px 1px var(--amino-gray-0) inset,
    var(--amino-button-focus-ring);
  --amino-surface-color: var(--amino-gray-0);
  --amino-surface-color-secondary: var(--amino-gray-50);
  --amino-backdrop-color: #000000;
  --amino-raised-surface-color: var(--amino-gray-0);
  --amino-raised-color-secondary: var(--amino-gray-50);
  --amino-button-primary-hover: linear-gradient(
    180deg,
    #446fee 0%,
    #2e55e2 100%
  );
  --amino-button-success-hover: linear-gradient(
    180deg,
    #37b04a 0%,
    #2ca041 100%
  );
  --amino-button-danger-hover: linear-gradient(
    180deg,
    #ee4559 0%,
    #dd314b 100%
  );
  --amino-button-warning-hover: linear-gradient(
    180deg,
    #ec6f23 0%,
    #e5611a 100%
  );
  --amino-button-standard-hover: linear-gradient(
    180deg,
    #ffffff 0%,
    #e8eaee 100%
  );
  --amino-spinner-track-black: rgba(0, 0, 0, 0.3);
  --amino-spinner-stroke-black: #0c0c0c;
  --amino-spinner-track-white: rgba(255, 255, 255, 0.3);
  --amino-spinner-stroke-white: #ffffff;
  --amino-appbar-height: 55px;
  --amino-sidebar-width: 232px;
  --amino-elevation-0: 0;
  --amino-elevation-100: 100;
  --amino-elevation-200: 200;
  --amino-elevation-300: 300;
  --amino-elevation-400: 400;
  --amino-elevation-500: 500;
  --amino-dialog-elevation: var(--amino-elevation-400);
  --amino-toast-elevation: var(--amino-elevation-500);
  --amino-appbar-elevation: var(--amino-elevation-500);
  --amino-size-xl: 56px;
  --amino-size-lg: 48px;
  --amino-size-md: 40px;
  --amino-size-sm: 32px;
  --amino-radius-4: 0.25rem;
  --amino-radius-6: 0.375rem;
  --amino-radius-8: 0.5rem;
  --amino-radius-10: 0.625rem;
  --amino-radius-12: 0.75rem;
  --amino-font-size-3xl: 1.875rem;
  --amino-font-size-2xl: 1.5rem;
  --amino-font-size-xl: 1.25rem;
  --amino-font-size-l: 1.125rem;
  --amino-font-size-base-docs: 1rem;
  --amino-font-size-base: 0.813rem;
  --amino-font-size-s: 0.75rem;
  --amino-font-size-xs: 0.688rem;
  --amino-line-height-3xl: 3rem;
  --amino-line-height-2xl: 2.5rem;
  --amino-line-height-xl: 2rem;
  --amino-line-height-l: 1.5rem;
  --amino-line-height-base: 1.5rem;
  --amino-line-height-s: 1rem;
  --amino-line-height-xs: 0.75rem;
  --amino-font-sans: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI',
    'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
    'Helvetica Neue', sans-serif;
  --amino-font-mono: Operator Mono, MonoLisa, Dank Mono, Menlo, Monaco,
    Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono,
    Courier New, monospace;
  --amino-font-harmonized-codes: 'Fira Code', var(--amino-font-sans);
  --amino-input-background: inherit;
}
"
`;

exports[`Case night theme: 1`] = `
"[data-theme='night'] {
  color-scheme: dark;
  --amino-glass-0: #ffffff00;
  --amino-glass-50: #ffffff0a;
  --amino-glass-100: #ffffff21;
  --amino-glass-200: #ffffff26;
  --amino-glass-300: #ffffff44;
  --amino-glass-400: #ffffff51;
  --amino-glass-500: #ffffff63;
  --amino-glass-600: #ffffff8e;
  --amino-glass-700: #ffffffa8;
  --amino-glass-800: #ffffffc1;
  --amino-glass-900: #ffffffd8;
  --amino-glass-950: #ffffffea;
  --amino-glass-1000: #fffffff4;
  --amino-gray-0: #0f1015;
  --amino-gray-50: #1e2029;
  --amino-gray-100: #272a35;
  --amino-gray-200: #343846;
  --amino-gray-300: #454c5e;
  --amino-gray-400: #555d72;
  --amino-gray-500: #68748d;
  --amino-gray-600: #8a95a8;
  --amino-gray-700: #bcc3cd;
  --amino-gray-800: #dfe2e7;
  --amino-gray-900: #e8eaee;
  --amino-gray-950: #f4f5f6;
  --amino-gray-1000: #ffffff;
  --amino-blue-50: #1a1f51;
  --amino-blue-100: #232e85;
  --amino-blue-200: #2438a9;
  --amino-blue-300: #2544d0;
  --amino-blue-400: #2e55e2;
  --amino-blue-500: #446fee;
  --amino-blue-600: #678ff3;
  --amino-blue-700: #98b6f8;
  --amino-blue-800: #c2d6fb;
  --amino-blue-900: #dae7fc;
  --amino-blue-950: #e7f0fe;
  --amino-cyan-50: #152d32;
  --amino-cyan-100: #1d464e;
  --amino-cyan-200: #25646f;
  --amino-cyan-300: #29818e;
  --amino-cyan-400: #2ca0af;
  --amino-cyan-500: #3ec4d0;
  --amino-cyan-600: #66d5db;
  --amino-cyan-700: #9be7e9;
  --amino-cyan-800: #c0f2f2;
  --amino-cyan-900: #d6fffe;
  --amino-cyan-950: #e6fefd;
  --amino-red-50: #49081a;
  --amino-red-100: #75142d;
  --amino-red-200: #9a1734;
  --amino-red-300: #b81835;
  --amino-red-400: #dd314b;
  --amino-red-500: #ee4559;
  --amino-red-600: #f67681;
  --amino-red-700: #faa7ab;
  --amino-red-800: #fccfd0;
  --amino-red-900: #fee5e5;
  --amino-red-950: #ffeceb;
  --amino-orange-50: #3f160b;
  --amino-orange-100: #763018;
  --amino-orange-200: #923c1a;
  --amino-orange-300: #b74a17;
  --amino-orange-400: #e5611a;
  --amino-orange-500: #ec6f23;
  --amino-orange-600: #f08e47;
  --amino-orange-700: #f5b27c;
  --amino-orange-800: #f9d3af;
  --amino-orange-900: #fdebd7;
  --amino-orange-950: #fef4e7;
  --amino-green-50: #0a2914;
  --amino-green-100: #1c4b29;
  --amino-green-200: #205b2f;
  --amino-green-300: #237234;
  --amino-green-400: #2ca041;
  --amino-green-500: #37b04a;
  --amino-green-600: #5dcb6a;
  --amino-green-700: #93e29a;
  --amino-green-800: #c3efc5;
  --amino-green-900: #e0f8e0;
  --amino-green-950: #f0faef;
  --amino-purple-50: #2f055c;
  --amino-purple-100: #4e1881;
  --amino-purple-200: #6c1eab;
  --amino-purple-300: #871ed2;
  --amino-purple-400: #a126ed;
  --amino-purple-500: #c052fa;
  --amino-purple-600: #d782fe;
  --amino-purple-700: #e9b2ff;
  --amino-purple-800: #f4d4ff;
  --amino-purple-900: #fae8ff;
  --amino-purple-950: #fcf0ff;
  --amino-transparent-white: #101116bb;
  --amino-transparent-black: #ffffffbb;
  --amino-transparent-gray-600: #94969e1f;
  --amino-v3-shadow-base: 0px 0px 0px 1px rgba(255, 255, 255, 0.16),
    0px 1px 4px 0px rgba(0, 0, 0, 0.7), 0px 1px 3px rgba(0, 0, 0, 0.6);
  --amino-v3-shadow-medium: 0px 0px 0px 1px rgba(255, 255, 255, 0.16),
    0px 2px 4px 0px rgba(0, 0, 0, 0.6), 0px 4px 6px rgba(0, 0, 0, 0.7);
  --amino-v3-shadow-large: 0px 0px 0px 1px rgba(255, 255, 255, 0.16),
    0px 4px 6px 0px rgba(0, 0, 0, 0.6), 0px 10px 15px rgba(0, 0, 0, 0.7);
  --amino-v3-shadow-xl: 0px 0px 0px 1px rgba(255, 255, 255, 0.16),
    0px 10px 10px 0px rgba(0, 0, 0, 0.6), 0px 20px 25px rgba(0, 0, 0, 0.7);
  --amino-v3-shadow-xxl: 0px 0px 0px 1px rgba(255, 255, 255, 0.16),
    0px 32px 50px 0px rgba(0, 0, 0, 0.7);
  --amino-v3-shadow-inset: 0px 2px 4px rgba(0, 0, 0, 0.24),
    0px 1px 0px rgba(17, 17, 20, 1), inset 0px 0px 8px rgba(0, 0, 0, 0.16),
    inset 0px 0px 0px 0.5px rgba(0, 0, 0, 0.24),
    inset 0px 2px 4px rgba(0, 0, 0, 0.24),
    inset 0px 1px 1px rgba(0, 0, 0, 0.24),
    0px 0px 6px rgba(255, 255, 255, 0.08),
    0px 0px 0px 1px rgba(255, 255, 255, 0.12);
  --amino-shadow-button-primary: 0px 1px 3px 0px rgba(68, 111, 238, 0.4),
    0px 0px 0px 1px rgba(255, 255, 255, 0.08) inset,
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset, 0px 0px 0px 0.5px #000;
  --amino-shadow-button-success: 0px 1px 3px 0px rgba(55, 176, 74, 0.4),
    0px 0px 0px 1px rgba(255, 255, 255, 0.08) inset,
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset, 0px 0px 0px 0.5px #000;
  --amino-shadow-button-danger: 0px 1px 3px 0px rgba(238, 69, 89, 0.4),
    0px 0px 0px 1px rgba(255, 255, 255, 0.08) inset,
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset, 0px 0px 0px 0.5px #000;
  --amino-shadow-button-warning: 0px 1px 3px 0px rgba(236, 111, 35, 0.4),
    0px 0px 0px 1px rgba(255, 255, 255, 0.08) inset,
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset, 0px 0px 0px 0.5px #000;
  --amino-button-focus-ring: 0 0 0 2px var(--amino-blue-500);
  --amino-shadow-button-standard: 0px 0px 0px 1px rgba(255, 255, 255, 0.08)
      inset,
    0px 1px 3px 0px rgba(0, 0, 0, 0.8),
    0px 0px 2px 1px rgba(255, 255, 255, 0.08) inset,
    0px -1px 1px 0px rgba(0, 0, 0, 0.4) inset,
    0px 0px 0px 0.5px rgba(0, 0, 0, 0.5);
  --amino-shadow-button-disabled: 0px 0px 0px 1px #000,
    0px -1px 1px 0px rgba(0, 0, 0, 0.2) inset,
    0px 0px 0px 1px rgba(255, 255, 255, 0.08) inset,
    0px 1px 3px 0px rgba(255, 255, 255, 0);
  --amino-shadow-select-base: 0px 0px 0px 1px #000,
    0px -1px 1px 0px rgba(0, 0, 0, 0.4) inset,
    0px 0px 2px 1px rgba(255, 255, 255, 0.08) inset,
    0px 1px 3px 0px rgba(0, 0, 0, 0.8),
    0px 0px 0px 1px rgba(148, 150, 158, 0.12) inset;
  --amino-shadow-select-active: 0px 0px 0px 1px #000,
    0px -1px 1px 1px rgba(114, 146, 243, 0.24) inset,
    0px 0px 0px 1px #7292f3 inset, 0px 1px 3px 0px rgba(114, 146, 243, 0.4);
  --amino-shadow-select-error: 0px 0px 0px 1px #000,
    0px -1px 1px 1px rgba(235, 92, 108, 0.24) inset,
    0px 0px 0px 1px #eb5c6c inset, 0px 1px 3px 0px rgba(235, 92, 108, 0.6);
  --amino-border: 1px solid var(--amino-border-color);
  --amino-border-blue: 1px solid var(--amino-blue-500);
  --amino-border-red: 1px solid var(--amino-red-500);
  --amino-border-cyan: 1px solid var(--amino-cyan-500);
  --amino-border-orange: 1px solid var(--amino-orange-500);
  --amino-border-green: 1px solid var(--amino-green-500);
  --amino-border-purple: 1px solid var(--amino-purple-500);
  --amino-border-amber: 1px solid var(--amino-amber-500);
  --amino-border-transparent: 1px solid transparent;
  --amino-border-subtle: 1px solid var(--amino-gray-100);
  --amino-sidebar-color: var(--amino-gray-0);
  --amino-header-color: var(--amino-gray-0);
  --amino-danger: var(--amino-red-500);
  --amino-danger-dark: var(--amino-red-700);
  --amino-error: var(--amino-red-500);
  --amino-primary: var(--amino-blue-500);
  --amino-primary-light: var(--amino-blue-400);
  --amino-primary-dark: var(--amino-blue-800);
  --amino-success: var(--amino-green-500);
  --amino-success-dark: var(--amino-green-700);
  --amino-warning: var(--amino-orange-500);
  --amino-warning-dark: var(--amino-orange-600);
  --amino-glow-error: 0 0 0 1px var(--amino-red-500);
  --amino-glow-blue: 0 0 0 1px var(--amino-primary);
  --amino-glow-cyan: 0 0 0 1px var(--amino-cyan-500);
  --amino-glow-red: 0 0 0 1px var(--amino-red-500);
  --amino-glow-orange: 0 0 0 1px var(--amino-orange-500);
  --amino-glow-green: 0 0 0 1px var(--amino-green-500);
  --amino-glow-purple: 0 0 0 1px var(--amino-purple-500);
  --amino-page-background: var(--amino-gray-0);
  --amino-border-color: var(--amino-gray-200);
  --amino-hover-color: var(--amino-gray-50);
  --amino-text-color: var(--amino-gray-1000);
  --amino-text-color-secondary: var(--amino-gray-700);
  --amino-focus-button-standard: 0px 0px 0px 1px
      var(--amino-transparent-gray-600) inset,
    var(--amino-button-focus-ring);
  --amino-focus-button: 0px 0px 0px 1px var(--amino-gray-0) inset,
    var(--amino-button-focus-ring);
  --amino-surface-color: var(--amino-gray-50);
  --amino-surface-color-secondary: var(--amino-gray-100);
  --amino-backdrop-color: #000000;
  --amino-raised-surface-color: #272a3599;
  --amino-raised-color-secondary: var(--amino-gray-0);
  --amino-button-primary-hover: linear-gradient(
    180deg,
    #446fee 0%,
    #2e55e2 100%
  );
  --amino-button-success-hover: linear-gradient(
    180deg,
    #37b04a 0%,
    #2ca041 100%
  );
  --amino-button-danger-hover: linear-gradient(
    180deg,
    #ee4559 0%,
    #dd314b 100%
  );
  --amino-button-warning-hover: linear-gradient(
    180deg,
    #ec6f23 0%,
    #e5611a 100%
  );
  --amino-button-standard-hover: linear-gradient(
    180deg,
    #272a3599 0%,
    #494c5780 100%
  );
  --amino-spinner-track-black: rgba(255, 255, 255, 0.3);
  --amino-spinner-stroke-black: #ffffff;
  --amino-spinner-track-white: rgba(0, 0, 0, 0.3);
  --amino-spinner-stroke-white: #0c0c0c;

  /* Fix for dark mode surface hover color. Dark mode surfaces have a lighter color than day mode inverted to see shadows better. */
  .elevated {
    --amino-hover-color: var(--amino-gray-100);
  }
}
"
`;
